v1/compat.go

```diff
@@ -0,0 +1,85 @@
+package machinery
+
+import (
+	"context"
+	"github.com/RichardKnop/machinery/v1/tasks"
```

Comment:
Please organize imports in 3 different groups:

- Library
- 3rd party
- Local
Same as described here: golang/tools@ed69e84

Reply:
Should be resolved, @marcocali

Reply:
No, it is not.